.github/workflows/build_test.yml

```diff
@@ -62,6 +62,8 @@ jobs:
 
       - name: Run E2E tests
         run: yarn test:run-cypress
+        env:
+          NODE_OPTIONS: "--no-experimental-require-module --no-experimental-detect-module" # See issue https://github.com/cypress-io/github-action/issues/1408
```

Comment:
Thanks for including the ref - much easier to delete things like this again if it's clear where to check.